### ISSUE TITLE
Dupe bug for Crafting Station

### ISSUE THREAD

**Issue by lllMohandarlll** [copygirl/BetterStorage#323]
The crafting station seems to duplicate some items like Terrafirmacraft food.

If you insert for example a piece of cheese of 160oz (full stack) with its native tags of:
<terrafirmacraft:item.Cheese>.withTag({foodWeight: 160.0 as float, foodDecay: -3.39 as float, decayTimer: 10456})

The Crafting Station return back to you a piece of cheese with these tags;

<terrafirmacraft:item.Cheese>.withTag({foodWeight: 999.0 as float, foodDecay: -21.2254 as float, decayTimer: 10455})

Could this be fixed by prohibit/disable the insertion of food in the crafting station?

I've tryed to make this through MineTweaker, but 
even with
mods.betterstorage.CraftingStation.remove
and
mods.betterstorage.CraftingStation.removeShapeless
and so on..
minetweaker still says that are not valid functions (so it's not a correct syntax)
isn't still possible at all?
minetweaker wiki says that "Each of these handlers can have recipes added or removed"

**Comment by copygirl** [comment]
That's unfortunate. I'm a big fan of TerraFirmaCraft but I dropped BetterStorage development / support a long time ago. I liked the Crafting Station and the way I made it work, though it's difficult to impossible to support every kind of crafting. Especially crafting hooks, which I guess is what breaks things in this case. One of my guesses is that the food stacks are being added up and modified in the crafting grid directly or so.

The MineTweaker wiki probably means that you can add and remove specific recipes, but I don't think you can blacklist regular crafting recipes that also work through the Crafting Station.

**Comment by lllMohandarlll** [comment]
Mhm, this is  sad :<
Well, I could suggest to look into the code of another mod called Avaritia eventually (if you have the wish and the time, naturally), because the avaritia crafting table for example doesn't allow other crafting to be made inside of it fo example, not sure btw if what I said have a sense at all.. meh
It would have been awesome if there was a config option to disable all the other crafting in the crafting station (except for the better storage one).. that would be a good solution.. unluckly I'm too poor on java skill to merge a new commit :l

**Comment by copygirl** [comment]
Since just adding a config setting for disabling Vanilla crafting in the crafting station is pretty easy, I compiled a new version *just for you~* ❤️ [Here is the new build.](https://github.com/copygirl/BetterStorage/releases/tag/v0.13.1.128)

Ideally of course, I'd fix the actual bug, which would require lots of debugging work and there's not even a guarantee that it can be fixed sensibly. The next better alternative is having a blacklist that disables certain Vanilla recipes, but even that is a bit more work than I'm currently willing to put into this technically abandoned mod.

**Comment by lllMohandarlll** [comment]
Yay! Thank you so much!!
This seems to work even with other recipes, they are disabled as well :D
So, I guess that I could just add through minetweaker the recipes that I truly want allow in the crafting station, right?
Btw sorry for the late in this reply, I've got and actually still have some trouble with the internet provider :/

**Comment by copygirl** [comment]
Yes, you can manually add those recipes again as crafting station specific recipes.

It's a bit sad, since you basically lose the autocrafting ability (if you were using that) for all Vanilla recipes unless you painfully add every one you want to use it for. But from the sounds of it you weren't so that's cool!

**Comment by lllMohandarlll** [comment]
Yeah don't worry ^^
I was using the crafting station as a custom crafting table to justify more advanced recipes (those that realistically talking could not be done without a proper support), and that support is the crafting station (with a new custom texture) :D
it's great to make a "progression" in technology :)

**Comment by copygirl** [comment]
> it's great to make a "progression" in technology :)

I love that part! Used it way back in my more difficult (early game) progression focused pack, myself.

Are you going to use the crafting time / experience cost requirement for your crafting recipes?

**Comment by lllMohandarlll** [comment]
Not really, at least, I would never do it upon vanilla exp since you lose it on death.. but you gave me some idea anyway, just in case.
Actually the system is just based on "common sense", with terrafirmacraft as the base, then you need the crafting station to begin to craft the mechanical components for flaxbear steampower, then you can do the rest on the rolling machine to unlock immersive, then you'll unlock the avaritia crafting table 9x9 where you can create tungsten components  for the more advanced mod (you know, tungsten it's an hard metal) :P

**Comment by copygirl** [comment]
Sounds like it might be right up my alley. Is it already available? Or can you let me know when it's out?

**Comment by lllMohandarlll** [comment]
Sure thing! :D I would be glad to hear your opinion about it!
It still has not an official release actually, I'm still working on it, but it has an alpha release, you can beta-test that anyway if you like :)
if you don't mind to add me on Discord it would be easier, I would prefer to not publicize the download link yet if possible
You can find me as Mohandar #3956
